enhancements/network/ovs-dynamic-cpu-pinning.md

```diff
@@ -0,0 +1,204 @@
+---
+title: ovs-dynamic-cpu-pinning
+authors:
+  - "@zeeke"
+reviewers:
+  - "@trozet"
+  - "@MarSik"
+  - "@ricky-rav"
+  - "@cgoncalves"
+approvers:
+  - "@trozet"
+api-approvers:
+  - "None"
+creation-date: 2023-03-31
+last-updated: 2023-03-31
+tracking-link:
+  - https://issues.redhat.com/browse/CNF-5977
+---
+
+# OVS Dynamic CPU Pinning
+
+## Summary
+
+Allow OVS daemons to run on reserved CPUs if the network workload requires
+cycles and non-reserved CPU are idle.
```

Comment:
So here it should be "allow... to run on reserved and non-reserved CPUs....", right?
@MarSik said it better in a previous comment:

> This enhancement should allow OVS to use cpus from outside of the system cpu reserved list while still obeying the restrictions and cpu reservations maintained by the cpu manager. In other words, OVS will be able to use extra cpus, but will not interfere with cpus assigned to containers in Guaranteed QoS pods.